config/mvi_schema/mvi_add_person_template.xml

```diff
@@ -0,0 +1,93 @@
+      <acceptAckCode code="AL"/>
+      <receiver typeCode="RCV">
+        <device determinerCode="INSTANCE" classCode="DEV">
+          <id root="2.16.840.1.113883.4.349"/>
+        </device>
+      </receiver>
+      <sender typeCode="SND">
+        <device determinerCode="INSTANCE" classCode="DEV">
+          <id extension="200VGOV" root="2.16.840.1.113883.4.349"/>
+        </device>
+      </sender>
+      <attentionLine>
+        <keyWordText>Search.Token</keyWordText>
+        <value xsi:type="ST">{{ search_token }}</value>
+      </attentionLine>
+      <controlActProcess classCode="CACT" moodCode="EVN">
+        <dataEnterer contextControlCode="AP" typeCode="ENT">
+          <assignedPerson classCode="ASSIGNED">
+            <id extension="{{ user_identity }}" root="2.16.840.1.113883.4.349"/>
+            <assignedPerson determinerCode="INSTANCE" classCode="PSN">
+              <name>
+                <given>{{ first_name }}</given>
+                <family>{{ last_name }}</family>
+              </name>
+            </assignedPerson>
+            <representedOrganization determinerCode="INSTANCE" classCode="ORG">
+              <id extension="dslogon.{{ edipi }}" root="2.16.840.1.113883.4.349"/>
+              <code code="{{ current_datetime }}"/>
+              <desc>vagov</desc>
+              <telecom value="{{ ip_address }}"/>
+            </representedOrganization>
+          </assignedPerson>
+        </dataEnterer>
+        <subject typeCode="SUBJ">
+          <registrationEvent classCode="REG" moodCode="EVN">
+            <id nullFlavor="NA"/>
+            <statusCode code="active"/>
+            <subject1 typeCode="SBJ">
+              <patient classCode="PAT">
+                <id extension="{{ user_identity }}" root="2.16.840.1.113883.4.349"/>
+                <statusCode code="active"/>
+                <patientPerson>
+                  <name use="L">
+                    <given>{{ first_name}}</given>
+                    <family>{{ last_name }}</family>
+                  </name>
+                  <birthTime value="{{ date_of_birth }}"/>
+                  <asOtherIDs classCode="SSN">
+                    <id extension="{{ ssn }}" root="2.16.840.1.113883.4.1"/>
+                    <scopingOrganization determinerCode="INSTANCE" classCode="ORG">
+                      <id root="2.16.840.1.113883.4.1"/>
+                    </scopingOrganization>
+                  </asOtherIDs>
+                  <asOtherIDs classCode="PAT">
+                    <id extension="PROXY_ADD^PI^200VBA^USVBA" root="2.16.840.1.113883.4.349"/>
+                    <scopingOrganization determinerCode="INSTANCE" classCode="ORG">
+                      <id extension="VBA" root="2.16.840.1.113883.4.349"/>
+                      <name>MVI.ORCHESTRATION</name>
+                    </scopingOrganization>
+                  </asOtherIDs>
+                </patientPerson>
+                <providerOrganization determinerCode="INSTANCE" classCode="ORG">
```

Comment:
This and the `custodian` element look like placeholder/sample values that we would want to populate correctly; are we getting any guidance about what values we are supposed to supply?

Reply:
That IS the guidance we were given. In fact, I checked with ebenefits just to see if they do anything differently in this regard and they fill it exactly the same way.